java/org/contikios/cooja/Cooja.java

```diff
@@ -3742,6 +3742,24 @@ private static File restoreContikiRelativePath(File portable) {
 
   private final static String PATH_CONFIG_IDENTIFIER = "[CONFIG_DIR]";
   public File currentConfigFile = null; /* Used to generate config relative paths */
+
+  /**
+   * Returns the config dir.
+   */
+  public String getConfigDir() {
+    if (currentConfigFile == null) {
+      return null;
+    }
+    return currentConfigFile.getParentFile().toString();
```

Comment:
It is unlikely with regard to how `currentConfigFile` is assigned but `getParentFile()` could return NULL. This is checked in other places in Cooja and if parent is NULL, the current directory is used instead.

Reply:
Fixed.